docs/howto_bridge.md

```diff
@@ -5,123 +5,104 @@ title: Bridge Assets
-* ETH: Native
-* DAI: 0x6B175474E89094C44Da98b954EedeAC495271d0F (always double check contract address)
+Use with caution. All crypto applications can potentially carry risks related to smart contracts/pallets.
 :::
 
 <div style={{textAlign: 'center'}}>
-  <img alt="metadata" src={useBaseUrl('/howto_bridge/inbound_2.jpg')} width="800px" />
+  <img alt="metadata" src={useBaseUrl('/howto_bridge/mb1.jpg')} width="600px" />
 </div>
 
-### 04 Select Payment for Gas
-* Following the network switch to Acala, select the payment method for gas **(6).** Note that the bridge minimum for feeless bridge (on Acala side) is ≥10 $DAI;
-* Click **Next** **(7)** to proceed. This will prompt Metamask to request a network switch back to Ethereum from Acala.
+### 02 Add the Wallets from Source and Destination Network
+
+- Once you have navigated to Carrier.so, add the wallets needed to allow for bridging to and from the desired networks **(1 in image above).**
+- In the example above, Metamask was selected as the wallet for Ethereum and Talisman for HydraDX.
 
 <div style={{textAlign: 'center'}}>
-  <img alt="metadata" src={useBaseUrl('/howto_bridge/inbound_3.jpg')} width="800px" />
+  <img alt="metadata" src={useBaseUrl('/howto_bridge/mb2.jpg')} width="600px" />
 </div>
 
-### Step 5: Bridge Tokens
-* Following the network switch back to Ethereum, click **Approve** **(8)** to proceed. This will prompt a  first transaction within your Metamask wallet to approve the tokens to be transferred;
-* Once this has been executed, click **Transfer** **(8)** (which will appear in place of Approve). This will prompt a second transaction within your Metamask wallet to execute the bridge transfer.
+### 03 Select Networks and Wallets to Bridge
 
 <div style={{textAlign: 'center'}}>
-  <img alt="metadata" src={useBaseUrl('/howto_bridge/inbound_4.jpg')} width="800px" />
+  <img alt="metadata" src={useBaseUrl('/howto_bridge/mb3.jpg')} width="600px" />
 </div>
 
-Once the transaction has been processed by Wormhole, click Redeem (9). This action results in you receiving the tokens on the Acala chain.
+- Once Metamask and Talisman are connected, select the network chains **(2)** and select the previously connected wallets **(3)**.
 
-<div style={{textAlign: 'center'}}>
-  <img alt="metadata" src={useBaseUrl('/howto_bridge/inbound_5.jpg')} width="800px" />
-</div>
+### 04 Select Asset to Bridge
 
-You are all set!
+- Select the token asset and amount of tokens you would like to bridge **(4)**.
 
-## Bridging Assets Acala -> ETH
-After transferring bridged assets back to Acala (using [cross-chain transfers](/howto_xcm)), you can bridge them back to the Ethereum ecosystem following
+### 05 Bridge Tokens
 
-### 01 Navigate to Wormhole Token Bridge Page
-[https://www.portalbridge.com/#/transfer](https://www.portalbridge.com/#/transfer)
+- Within **Settings** **(5)**, you can select whether to Auto Relay the transaction.
```

Comment:
Shouldn't this step be urging the user to select this? otherwise I'm guessing they won't be auto routed to HydraDX?